BlazarBase/src/main/java/com/hubspot/blazar/base/BuildCommand.java

```diff
@@ -0,0 +1,61 @@
+package com.hubspot.blazar.base;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Objects;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Lists;
+
+public class BuildCommand {
+  private static final String DEFAULT_CMD = "/bin/bash";
+  private static final String DEFAULT_ARG = "-c";
```

Comment:
Its not part of the default command if you override a field, I'll go make it do that if you didn't set the executable field.